README.md

```diff
@@ -191,6 +223,15 @@ Inventory file example for CentOS/RHEL and Fedora systems.
     192.168.22.25 vm_name=fastvm-rhel-8.0-25 ansible_python_interpreter=/usr/libexec/platform-python
     192.168.22.26 vm_name=fastvm-rhel-8.0-26 ansible_python_interpreter=/usr/libexec/platform-python
 
+Inventory file example with two full members and two remote nodes:
+
+    [cluster-full]
+    192.168.22.21 vm_name=fastvm-centos-7.6-21
+    192.168.22.22 vm_name=fastvm-centos-7.6-22
+    [cluster-remote]
+    192.168.22.23 vm_name=fastvm-centos-7.6-23 cluster_is_remote=True
+    192.168.22.24 vm_name=fastvm-centos-7.6-24 cluster_is_remote=True
```

Comment:
- [ ] if the cluster nodes and remote nodes are in different groups the remote nodes are not added. I have tried that remote nodes are added well when together in one group with normal nodes like this:
~~~
    [cluster-with-remotes]
    192.168.22.21 vm_name=fastvm-centos-7.6-21
    192.168.22.22 vm_name=fastvm-centos-7.6-22
    192.168.22.23 vm_name=fastvm-centos-7.6-23 cluster_is_remote=True
~~~

Reply:
Are you sure? I was testing with hosts in different groups. This role should not care which groups they are in - only whether cluster_is_remote is true or false.

Reply:
what have you used for 'hosts: xxx' in playbook? It sounds as if using 'all' when it doesn't care about groups and makes all hosts into single one.

Reply:
Both full and remote host should be targeted hosts. In the above example you could do this using `hosts: all`, `hosts: cluster-full:cluster-remote`, or make a top level group including both subgroups.

Reply:
Thank you for clarification. I know it is possible to separate hosts in multiple groups, but in case here what is then the reason for separating the remote nodes and normal nodes into separate groups?
In my opinion the single group looks more consistent with rest of README but I can imagine keeping two separate groups if necessary if the example playbook mentioning both groups (`hosts: cluster-full:cluster-remote`) is also added. Would it be possible to adjust the in one of these two ways? Or if you have some other idea also please let me know.

Reply:
Added all nodes to one group.